src/content/docs/en/hardware/requirements.mdx

```diff
@@ -5,34 +5,34 @@ description: "Minimum Requirements for Ultramarine Linux"
 
 import Alert from "../../../../components/Docs/Alert.astro";
 
-## Recommended Requirements for Standard PCs
+## Recommended Specifications for Standard PCs
 
 These are our suggestions for the most basic (daily-driveable) system:
 
 - 4GB of RAM (8GB or more recommended)
-- 64-bit processor (x86_64/amd64)
-- 16GB of free disk space (24GB or more recommended)
+- Dual Core 64-bit processor (x86_64/amd64)
+- 20GB of free disk space (32GB or more recommended)
 - Internet connection (for downloading the ISO image, updates, and additional drivers)
 - A computer that supports booting from a USB drive
 - A USB drive (for creating the bootable media)
 
 ## Minimum Requirements for Raspberry Pi
 
 - Raspberry Pi 3 or newer
-- 4GB of RAM **OR** 2GB of RAM without a desktop environment
-- 9GB Class 10 MicroSD Card
+- 4GB of RAM **OR** 1GB of RAM without a desktop environment
+- 16GB Class 10 MicroSD Card
 - See the [anywhere page](/en/anywhere/rpi)
 
 ## Minimum Requirements for ARM Devices Using UEFI
 
 - 64-bit processor (aarch64)
-- 4GB or RAM
+- 4GB of RAM
 - 16GB of Storage
 
 ## Minimum Requirements
```

Comment:
This entire section should be moved up above rpi.

Reply:
bet